sdk/src/test/scala/com/horizen/consensus/ConsensusValidatorOmmersTest.scala

```diff
@@ -24,7 +24,7 @@ class ConsensusValidatorOmmersTest
 
   val consensusValidator = new ConsensusValidator {
     // always successful
-    private[horizen] override def verifyVrfProof(history: SidechainHistory, verifiedBlock: SidechainBlockHeader): Unit = {}
+    private[horizen] override def verifyVrfProof(history: SidechainHistory, verifiedBlock: SidechainBlockHeader, vrfMessage: VrfMessage): Unit = {}
```

Comment:
FYI: In case if you'll have something broken in Ommers tests - don't care. Can just ignore that test.
In Ommers_validation fixes and differences will be present.